models/Document.php

```diff
@@ -653,7 +653,7 @@ public function getDependencies()
      */
     public function setChildren($children)
     {
-        $this->children = $children;
+        $this->children[0] = $children;
```

Comment:
What if $children contains unpublished ones or is this just for internal use ?

Reply:
default index for internal use